test/jdk/tools/launcher/ExecutionEnvironment.java

```diff
@@ -24,49 +24,27 @@
 /*
  * @test
  * @bug 4780570 4731671 6354700 6367077 6670965 4882974
- * @summary Checks for LD_LIBRARY_PATH and execution  on *nixes
- * @requires os.family != "windows" & !vm.musl & os.family != "aix"
+ * @summary Checks for LD_LIBRARY_PATH and execution on *nixes
  * @library /test/lib
  * @modules jdk.compiler
  *          jdk.zipfs
  * @compile -XDignore.symbol.file ExecutionEnvironment.java
```

Comment:
One addition as for cleaning: I'm wondering whether we need this @compile directive with `-XDignore.symbol.file`. I guess this can be removed, too.